### PR TITLE
Sync CDOT dev branch with changes in USDOT develop

### PR DESCRIPTION
**Pull request by dmccoystephenson**
The purpose of this PR is to merge the latest changes in USDOT's develop branch into CDOT's dev branch. This addresses some merge conflicts for https://github.com/usdot-jpo-ode/jpo-security-svcs/pull/25.

### CODE REVIEW

**Comment by drewjj** [comment]
Any idea why this is doing CI checks while other pull requests on this repo didn't?

**Comment by drewjj** [comment]
> Any idea why this is doing CI checks while other pull requests on this repo didn't?

Aw! It added a ci.yml. We should probably make sure this is running y21 like the project.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+name: CI
+on: 
+  pull_request:
+  push:
+    
+jobs:
+  jpo-security-svcs:
+   runs-on: ubuntu-latest
+   container:
+     image: openjdk:17-jdk-slim-buster
```
Comment (drewjj) [inline .github/workflows/ci.yml]: Probably want to use jdk 21.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+   steps:
+     - name: Checkout ${{ github.event.repository.name }}
+       uses: actions/checkout@v3    
+     - name: Set up Maven
+       uses: stCarolas/setup-maven@v4.5
+       with:
+        maven-version: 3.8.2          
+     - uses: actions/setup-java@v3
+       with:
+         java-version: '11'
```
Comment (drewjj) [inline .github/workflows/ci.yml]: Probably want to specify 21.

**Review by drewjj — CHANGES_REQUESTED** [review]
Looks good besides these two things.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dmccoystephenson) [reply .github/workflows/ci.yml]: This has been changed to maven:3.8-eclipse-temurin-21-alpine

File: .github/workflows/ci.yml (hunk 2 above)
Comment (dmccoystephenson) [reply .github/workflows/ci.yml]: This has been changed to '21'

**Review by drewjj — APPROVED** [review]
I guess we will let Leidos figure out the CI stuff if they want it.

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**